.gitignore

```diff
@@ -1,19 +1,39 @@
 # OSX
 .DS_Store
-.idea/
-
-# VS Code
-.vscode
 
 # Vagrant
 .vagrant/
 ubuntu-xenial-16.04-cloudimg-console.log
 
+# don't commit contract binaries in protocol
+# we do commit the contract binaries in platform
```

Comment:
this comment seems from outdated code? is it needed?

Reply:
Agree. Done.